### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by henrikgrundmann**
Transfer customer_lifetime_value export to export service
Transfer rebuy_rate export to export service
Check which days to be reported on are still missing from the database
Compute values for those days
Write the results to DB, day by day
If table Does not exist in DB, create it accoring to specified sql

### CODE REVIEW

File: airflow.cfg [hunk 1]
```diff
@@ -0,0 +1,351 @@
+[core]
```
Comment (carolinux) [inline airflow.cfg]: I think it makes sense to delete the airflow config file from this PR after all.
I will include it in my PR that does the system scripts & setup.

File: aux/rebuy_rate_io.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
+""" Methods that help reading and writing from the results database"""
+#from collections import OrderedDict
+import os
```
Comment (carolinux) [inline aux/rebuy_rate_io.py]: rename this to `db_io.py` as we discussed

File: aux/rebuy_rate_io.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+import pandas as pd
+from sqlalchemy import create_engine, MetaData
+from sqlalchemy.sql import select, text
+#from sqlalchemy.orm import sessionmaker, scoped_session
+
+from aux.datetools import get_date_range_list
+import config
+
+#def get_session():
```
Comment (carolinux) [inline aux/rebuy_rate_io.py]: we can safely remove the commented out code

File: aux/rebuy_rate_io.py [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+        
+        
+def get_table(table_name, connection):
+    """Get the full table description from the database (column names and types)"""
+    m = MetaData()
+    m.reflect(connection)
+    return m.tables[table_name]
+
+    
+#def write_result_long(date, version, series, table_name):
```
Comment (carolinux) [inline aux/rebuy_rate_io.py]: remove commented out code

File: aux/rebuy_rate_io.py [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+        select(columns = ['*'], from_obj = text(table_name))
+            .where(text("date = '{0}' AND version = '{1}'".format(date_str, version)))
+    )
+
+    rows = [row for row in result]
+    connection.close()
+    return len(rows) > 0
+
+
+def can_cast_as_float(value):
```
Comment (carolinux) [inline aux/rebuy_rate_io.py]: this method is not used now (was used for csv exports before), remove

File: config.py [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+"""
+This module stores the filepaths for our various scripts as dictionaries
+"""
+
+basepath = '/home/grundmann/hgrundmann_thor/bi-project/'
```
Comment (carolinux) [inline config.py]: get this from an env var

File: airflow.cfg (hunk 1 above)
Comment (henrikgrundmann) [inline airflow.cfg]: done

File: aux/rebuy_rate_io.py (hunk 2 above)
Comment (henrikgrundmann) [inline aux/rebuy_rate_io.py]: done

File: aux/rebuy_rate_io.py (hunk 3 above)
Comment (henrikgrundmann) [inline aux/rebuy_rate_io.py]: done

File: aux/rebuy_rate_io.py (hunk 4 above)
Comment (henrikgrundmann) [inline aux/rebuy_rate_io.py]: done

File: aux/rebuy_rate_io.py (hunk 5 above)
Comment (henrikgrundmann) [inline aux/rebuy_rate_io.py]: done

File: config.py (hunk 6 above)
Comment (henrikgrundmann) [inline config.py]: done

File: config.py [hunk 7]
```diff
@@ -3,7 +3,9 @@
 This module stores the filepaths for our various scripts as dictionaries
 """
 
-basepath = '/home/grundmann/hgrundmann_thor/bi-project/'
+import os
+
+basepath = os.environ['BI_REPO_BASEPATH=']
```
Comment (carolinux) [inline config.py]: remove the equals sign

Comment (henrikgrundmann) [inline config.py]: done

**Pull request merged**